src/applications/post-911-gib-status/utils/helpers.jsx

```diff
@@ -318,95 +286,95 @@ export const wizardConfig = [
           <div>
             <p>
-              If you’re having trouble accessing your benefit statement, it
-              could be for one of these reasons:
+              The GI Bill benefit statement isn't available online to family
+              members and dependents. You'll need to request a new Certificate
+              of Eligibility letter to check your GI Bill benefit status.
             </p>
-            <ul>
-              <li>
-                We’re still processing your education benefits application and
-                we haven’t yet created a record for you. We usually process
-                applications within 60 days. If you applied less than 60 days
-                ago, please check back soon.
-              </li>
-              <li>
-                The name on your account doesn’t exactly match the name we have
-                in our Post-9/11 GI Bill records.
-              </li>
-              <li>
-                You haven’t yet applied for Post-9/11 GI Bill education
-                benefits.
-              </li>
-            </ul>
             <p>
-              If none of these situations apply to you, and you think your
-              Statement of Benefits should be here, please call the Education
-              Call Center at 888-442-4551 (888-GI-BILL-1). We’re here Monday
-              through Friday, 8:00 a.m. to 7:00 p.m. ET.
+              To request a COE, please call the Education Call Center at
+              888-442-4551 (888-GI-BILL-1). We’re here Monday through Friday,
+              8:00 a.m. to 7:00 p.m. ET.
             </p>
           </div>
         }
         status="warning"
       />
     ),
-    isActive: choices =>
-      choices.veteran ||
-      choices.automaticEligibility ||
-      choices.benefitsTransferred,
+    isActive: previousValue => previousValue === false,
   },
   {
-    type: 'transferMessage',
-    previous: 'benefitsTransferred',
+    type: 'errorMessage',
+    previous: 'veteran',
     component: () => (
       <AlertBox
+        headline="We’re sorry. We still can’t find your Post-911 GI Bill benefit statement"
         content={
-          <p>
-            Your sponsor needs to transfer their Post-9/11 GI Bill benefits to
-            you before we can process your application.
-          </p>
+          <div>
+            <p>
+              If you’re having trouble accessing your benefit statement, please
+              call the Education Call Center at 888-442-4551 (888-GI-BILL-1).
+              We’re here Monday through Friday, 8:00 a.m. to 7:00 p.m. ET.
+            </p>
+          </div>
         }
         status="warning"
       />
     ),
-    isActive: previousValue => previousValue === false,
+    isActive: previousValue => previousValue === true,
   },
 ];
+
+export function authenticationErrorMessage() {
+  return (
+    <div className="vads-u-margin-bottom--2">
+      {/* <div className="medium-8 columns vads-u-margin-bottom--2"> */}
```

Comment:
Does this need to stay here?

Reply:
good catch. I was copy-pasting similar things and then realized those extra classes weren't needed